src/main/java/UI/UI.java

```diff
@@ -0,0 +1,18 @@
+package UI;
+
+/**
+ * print messages with horizontal line
+ */
+public class UI {
+    protected static final String HORI_LINE = "---------------------------";
+    protected String content;
+
+    public UI() {
+    }
+
+    public void printMsg(String message) {
+        this.content = message;
```

Comment:
I don't think `content` is necessary as a variable? Can just `System.out.println(message)`.